components/MonogramAvatar/MonogramAvatar.jsx

```diff
@@ -68,7 +76,7 @@ class MonogramAvatar extends React.Component<Props, State> {
                 disableLink
                 onError={this.handleImageLoadError}
                 showPopover={false}
-                size={'base'}
+                size={this.props.size}
```

Comment:
Since you are deconstructing this, just `size` will work.

Reply:
That didn't seem to work. That's not a deconstruction, is it?

Reply:
It is... the destructuring of `size` happens at line `58`. You should be able to do a `size={size}` at line `78`.